package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "weather-app-backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "jest",
+    "start": "node dist/server.js",
+    "build": "tsc"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.19.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.24.4",
+    "@babel/preset-env": "^7.24.4",
+    "@babel/preset-typescript": "^7.24.1",
+    "@types/cors": "^2.8.17",
+    "@types/express": "^4.17.21",
+    "@types/jest": "^29.5.12",
+    "@types/node": "^20.12.7",
+    "@types/supertest": "^6.0.2",
+    "jest": "^29.7.0",
+    "supertest": "^6.3.4",
+    "ts-jest": "^29.1.2",
+    "typescript": "^5.4.5"
```

Comment:
do you think it is in the right place?

Reply:
It was automatically created when I initialized the project with git init -y. Many open-source projects, especially those in the Node.js and JavaScript ecosystem, use the ISC License due to its simplicity and compatibility with other licenses, being an open-source software license that allows for the free use, modification, and distribution of the software under certain conditions. Users are granted the freedom to use, modify, and distribute the software with the requirement to include the original copyright notice in any distributions.

Reply:
These are packages that are only required during development. They are used for tasks like compiling TypeScript code, running tests. This is why they are installed as DevDependencies. Also, I searched online and usually the best practices are to install them as devdependencies. This was my understanding, but if I got it wrong, please advice. :)